versioned_docs/version-8.6/reference/data-collection/data-collection.md

```diff
@@ -0,0 +1,119 @@
+
+### Usage telemetry data (SaaS and Desktop Modeler only)
+
+Usage telemetry data includes limited product usage data to help make better product improvement decisions and enable outreach to support users. This section describes the types of data and how they are collected.
+
+- Feature Usage:
+
+  - SaaS System Actions: All SaaS organizations submit basic information about which features are being used as part of telemetry data collection. When certain features are used, Camunda logs which feature is used and basic information about how it has been used. This information is tied to a pseudonymized organization.
+  - SaaS User Actions: Users that opt-in to analytics cookies gain access to in-app tutorials and automatically submit information about which features they interact with in Camunda’s UI as part of telemetry data. In addition to the data collected from system actions described above, Camunda collects cursor activity, geographical area, browser information, and basic biographical information limited to email, name, and city/region/country for user actions. If a user interacts through API, then personal information is not collected.
+
+  <img src={CookiePrefsImg} alt="Cookie preferences in user settings" width="300px" style={{border: 'none', padding: '0', marginTop: '0', backgroundColor: 'transparent'}}/>
+  <img src={AnalyticsImg} alt="Analytics opt-in menu" width="500px" style={{border: 'none', padding: '0', marginTop: '0', backgroundColor: 'transparent'}}/>
+
+  - Desktop Modeler User Actions: Users opting into collection of telemetry data in [Desktop Modeler](/components/modeler/desktop-modeler/telemetry/telemetry.md) send data to Camunda when to track how certain features are used, as described in the linked document.
+
+- AI Usage: Camunda's AI features, currently available in SaaS only, are clearly labeled as AI features. For Enterprise organizations, these features must be enabled by the customer via opt-in in the [Console](/components/console/manage-organization/enable-alpha-features.md#enable-ai-powered-features). Depending on the feature, they may collect different information.
+  - Camunda [Docs AI](/components/modeler/web-modeler/advanced-modeling/camunda-docs-ai.md) records the entire conversation to provide ongoing support.
+  - Camunda [Copilots](/docs/components/early-access/alpha/bpmn-copilot/bpmn-copilot.md) only gather usage telemetry data. Camunda automatically logs all information sent to and from our AI models for system monitoring by a limited set of operators. Camunda will only use the data from free users for product and model improvement.
```

Comment:
@HanselIdes @akeller We have no concept of alpha features in the 8.6 documentation, so to add this back in, I've linked to the /next version of the docs here.